lib/Models/createCatalogItemFromFileOrUrl.ts

```diff
@@ -166,6 +166,7 @@ function getConfirmation(
 function loadItem(newCatalogItem: BaseModel, fileOrUrl: File | string) {
   if (typeof fileOrUrl === "string") {
     newCatalogItem.setTrait(CommonStrata.definition, "url", fileOrUrl);
+    newCatalogItem.setTrait(CommonStrata.user, "url", fileOrUrl);
```

Comment:
Yeah, at the moment only the user strata is shared so setting the url at the user stratum seemed like the most straight-forward thing to do (otherwise the url would probably need to be shared separate to the user stratum). At the same time, I didn't want to accidentally mess something else up by not setting url in definition strata which is why I left that there.